src/System.Linq/src/System/Linq/DebugView.cs

```diff
@@ -12,7 +11,10 @@ namespace System.Linq
     // NOTE: DO NOT DELETE THE FOLLOWING DEBUG VIEW TYPES.
     // Although it might be tempting due to them not be referenced anywhere in this library,
     // Visual Studio currently depends on their existence to enable the "Results" view in 
-    // watch windows.
```

Comment:
It's possible, but we know for certain that Visual Studio does, e.g.
https://github.com/dotnet/roslyn/blob/a4e375b95953e471660e9686a46893c97db70b0e/src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/TypeHelpers.cs#L583